src/request/import/ImportFile.js

```diff
@@ -2,7 +2,7 @@
 /* global Nimiq */
 /* global Key */
 /* global ImportWords */
-/* global FileImport */
+/* global FileImporter */
```

Comment:
why the name change?

Reply:
I found the names `ImportFile` and `FileImport` to be too similar and confusing, thus I made this component into the `FileImporter`.